Index: pytest-mpi/docs/conftest.py
===================================================================
--- pytest-mpi.orig/docs/conftest.py	2022-03-17 01:18:33.457636933 +0100
+++ pytest-mpi/docs/conftest.py	2022-04-02 10:13:21.086405965 +0200
@@ -3,7 +3,10 @@
 import pytest
 
 from sybil import Sybil
-from sybil.parsers.codeblock import PythonCodeBlockParser
+try:     # sybil >= 3
+    from sybil.parsers.codeblock import PythonCodeBlockParser
+except ImportError:  # sybil < 3
+    from sybil.parsers.codeblock import CodeBlockParser as PythonCodeBlockParser
 from sybil.parsers.doctest import DocTestParser
 from sybil.parsers.skip import skip
 
